.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: GitHub CI
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu-build-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install deps
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y libevent-dev libseccomp-dev git libsasl2-dev
+    - name: Build
+      run: |
+        gcc --version
+        ./autogen.sh
+        ./configure --enable-seccomp --enable-tls --enable-extstore --enable-sasl --enable-sasl-pwdb
+        make -j
+    - name: Test
+      run: make test
+    - name: Test TLS
+      run: make test_tls
+
+  macos-build-test:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install deps
+      run: |
+        brew install automake
+        export OPENSSL_PREFIX=/usr/local/opt/openssl@1.1
+        export LD_LIBRARY_PATH="${OPENSSL_PREFIX}/lib"
+        cpan Net::SSLeay
+        cpan IO::Socket::SSL
+    - name: Build
+      run: |
+        clang --version
+        ./autogen.sh
+        export CC=clang
+        export ac_cv_libssl_dir=/usr/local/opt/openssl@1.1
+        ./configure --enable-tls --enable-extstore
```

Comment:
--enable-extstore is the default.

Reply:
> OS X is always clang?

I intentionally use clang since gcc is already tested on Ubuntu so it's like testing both on macOS and against clang compiler. What do you think? Do we need to change to gcc?

Reply:
I'd rather keep the delta to a minimum, but if that's the default compiler (I can't remember anymore) that'll have to be I guess? I'll need to get another buildbot with clang up since I don't want this thing I don't have access to be "the bug is either clang or OS X"

Reply:
I'll just remove explicitly setting clang. I think default is gcc.